specs/host_list_view.adoc

```diff
@@ -0,0 +1,148 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of hosts discovered by Tendrl
+
+Display list of hosts discovered by Tendrl, whether they currently associated with a cluster or not. The list should have functionality for 
+
+* filtering by name, cluster, role and status.
+* sorting by name.
+
+Host list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of hosts discovered by Tendrl along with their details.
+
+The details shown for each host in UI is -
+
+* Host Name
+* Status
+* IP Address
+* Storage Usage
+* CPU Utilisation
+* Memory Usage
+* Cluster Name
+* Role
```

Comment:
Would it be possible, that node will have more than one role and do we count with this situation?